docs/commands/zed.md

```diff
@@ -574,6 +570,28 @@ API for automation.
 
 > Note that the branchlog meta-query source is not yet implemented.
 
+### Manage
+```
+zed manage [options]
+```
+Maintenance tasks are performed on a lake with the `manage` command.
+
+Currently the only supported task is _compaction_, which reduces fragmentation
+by reading data objects in a pool and writing them back globally sorted and
+with their contents stored in contiguous large objects.
```

Comment:
I don't think "globally sorted" and "contiguous" are accurate descriptions of what's going on here. How about "non-overlapping"?

```suggestion
by reading data objects in a pool and writing their contents back to large,
non-overlapping objects.
```

Reply:
@nwt: Thanks for this suggestion. Happy to take it. In case it wasn't noticed, the wording I was using was taken directly from text that I was removing elsewhere in this PR, i.e., what I'm quoting below was an earlier forward-looking statement that's still present right now in our docs but will be removed if this PR merges. Now that compaction has been implemented I certainly recognize our way of talking about it may have evolved, but that was not obvious to me.

> Note that data is easily compacted by reading from a fragmented pool and writing it back to a target pool so that it is globally sorted and compacted into contiguous large objects.

Reply:
@philrz: I hadn't realized that.